components/builder-api/src/server/resources/jobs.rs

```diff
@@ -244,10 +246,10 @@ fn do_group_promotion_or_demotion(
     let channel = match Channel::get(origin, channel, &*conn) {
         Ok(channel) => channel,
         Err(NotFound) => {
-            if channel != STABLE_CHANNEL || channel != UNSTABLE_CHANNEL {
+            if channel != &ChannelIdent::stable() || channel != &ChannelIdent::unstable() {
```

Comment:
Merge conflict here with the fix to change to "&&" that has gone in..

Reply:
Oops, I fixed it earlier, but hadn't force-pushed the rebase. Done now!